### ISSUE TITLE
help request: 2.8.0 helm chart not released?

### ISSUE THREAD

**Issue by L1ghtman2k** [fluent/fluent-operator#1150]
### Describe the issue

Unsure if this was intended, but I noticed that the 2.8.0 helm chart wasn't released

**Comment by misoknr** [comment]
I noticed as well. Also, the link to the chart breaks the convention with the suffix "-2.8.0". All previous versions the file name is just "fluent-operator.tgz" since the version is in the URL.

https://github.com/fluent/fluent-operator/releases/download/v2.8.0/fluent-operator-2.8.0.tgz

**Comment by lmontanag** [comment]
Hi Folks , any idea on this one 

latest chart released for `fluent/fluent-operator` is `2.7.0`

```bash
helm repo add fluent https://fluent.github.io/helm-charts
helm repo update
```

```bash
helm search repo fluent
NAME                  	CHART VERSION	APP VERSION	DESCRIPTION
fluent/fluent-bit     	0.46.7       	3.0.4      	Fast and lightweight log processor and forwarde...
fluent/fluent-operator	2.7.0        	2.7.0      	Fluent Operator provides great flexibility in b...
fluent/fluentd        	0.5.2        	v1.16.2    	A Helm chart for Kubernetes
```

**Comment by berowil** [comment]
Would it be possible to release 2.8.0 in the near future? :)
